### PR TITLE
Nina geodesics h2

### PR DESCRIPTION
**Pull request by ninamiolane**
Add plot of geodesics for the hyperbolic space in 2D, with Poincare disk visualization.
Add unit tests to check that the geodesics belong to the space for all spaces.

### CODE REVIEW

File: tests/test_hyperbolic_space.py [hunk 1]
```diff
@@ -303,6 +303,21 @@ def test_exp_and_dist_and_projection_to_tangent_space(self):
         expected_1 = math.sqrt(sq_norm)
         self.assertTrue(np.allclose(result_1, expected_1))
 
+    def test_geodesic_and_belongs(self):
+        # TODO(nina): this tests fails when geodesic goes "too far"
```
Comment (johmathe) [inline tests/test_hyperbolic_space.py]: that's gonna be interesting to fix :)

File: examples/plot_geodesics_h2.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+"""
+Plot a geodesic on the hyperbolic space h2,
```
Comment (johmathe) [inline examples/plot_geodesics_h2.py]: could you potentially add one or a few renderings to this or a subsequent CL? I'm curious :)

File: geomstats/visualization.py [hunk 3]
```diff
@@ -153,20 +180,34 @@ def plot(points, ax=None, space=None, **point_draw_kwargs):
...
         trihedrons = convert_to_trihedron(points, space=space)
         for t in trihedrons:
             t.draw(ax, **point_draw_kwargs)
+
     elif space is 'S2':
         sphere = Sphere()
         sphere.add_points(points)
         sphere.draw(ax)
 
+    elif space is 'H2':
```
Comment (johmathe) [inline geomstats/visualization.py]: are you raising some ValueError if space is not in the 4 possibilities that you're passing?

File: examples/plot_geodesics_h2.py (hunk 2 above)
Comment (ninamiolane) [reply examples/plot_geodesics_h2.py]: screen shot 2018-02-21 at 11 39 52 am

The curvature increases next to the disk center and the points get closer as the geodesic approches the disk boundary, which makes sense as the disk's boundary represents the infinity, the boundary is never reached.

**Pull request merged**